hepcrawl/spiders/base_spider.py

```diff
@@ -188,12 +200,17 @@ def build_item(self, response):
         record.add_value('files', response.meta.get("direct_link"))
         record.add_value('urls', response.meta.get("urls"))
         record.add_xpath('abstract', './/dc:description/text()')
-        record.add_xpath('title', './/dc:title/text()')
+        title, subtitle = self.get_title(node)
+        if title:
+            record.add_value('title', title)
+        if subtitle:
+            record.add_value('subtitle', subtitle)
         record.add_xpath('date_published', './/dc:date/text()')
         record.add_xpath('source', './/base_dc:collname/text()')
         record.add_value("authors", self.get_authors(node))
         record.add_value('thesis', {'degree_type': 'PhD'})
-        record.add_value('collections', ['HEP', 'THESIS'])  # When it is checked if a thesis is about HEP?
+        # NOTE: When it is checked if a thesis is about HEP:
```

Comment:
We can assume that a thesis for INSPIRE is always HEP. However, indeed from base there might be non-hep thesis. We'll have selective harvests combined with automatic classification (e.g. auto-reject) in the workflow on INSPIRE.